main.py

```diff
@@ -1,10 +1,51 @@
+import os.path
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    directory = os.path.join("db", "tests", "players.json")
+
+    with open(directory, "r") as data_players:
+        players = json.load(data_players)
```

Comment:
Also you can just write:
with open("players.json") as data_players
And it will works without directoty variable